Makefile

```diff
@@ -8,13 +8,24 @@ BENCHTIME ?= 1x
 BENCHTIMEOUT ?= 20m
 BENCHCOUNT ?= 1
 
+podman =
+# docker --version might not contain any traces of podman in the latest
+# version, search for more output
 ifeq (,$(findstring podman,$(shell docker --version 2>/dev/null)))
```

Comment:
Isn't this
```make
ifeq (,$(findstring podman,$(shell docker --version 2>/dev/null)))
```
reverse of what it should be?

IIUC, `findstring` when found returns what it found. I.e. if "podman" is mentioned, `findstring` will return "podman".
However, `ifeq` here proceeds only when an empty string is the result of `findstring`.

If you check the former code, `DOCKER_OPTS=--user "$(shell id -u)"` thing applied to `docker`, not `podman`.

Reply:
Right, I've got fooled by the fact that this combination was still return the expected value, but due to a wrong reasons.